roles/lib/files/FWO.Data/Middleware/NormalizedConfigParameters.cs

```diff
@@ -0,0 +1,15 @@
+
+using System.Text.Json.Serialization; 
+using Newtonsoft.Json;
+
+namespace FWO.Data.Middleware
+{
+    public class NormalizedConfigGetParameters
+    {
+        [JsonProperty("mgm-ids")]
+        public int[] ManagementIds { get; set; } = [];
+
+        [JsonProperty("config=time")]
```

Comment:
config=time looks weird - can you explain this to me?

Reply:
fixed this typo